app/controllers/admin_controller.rb

```diff
@@ -0,0 +1,18 @@
+class AdminController < ApplicationController
+  before_action :logged_in_user, :require_admin
+
+  def require_admin
+    return if current_user&.admin?
```

Comment:
```suggestion
    return if current_user.admin?
```
ko cần toán từ an toàn, vì ở đây em đã check user đã login nè 
nên ở đây current user sẽ khác nil rồi